vaadinfrontend/frontend/themes/datamanager/components/card.css

```diff
@@ -16,6 +16,7 @@
   justify-content: center;
   padding: 1.5rem;
   flex: auto;
+  max-width: 300px;
```

Comment:
do we want concrete pixel values? Might look completely different based on the screen resolution

Reply:
I tried to look into this but find conflicting best practices. 
The jist is we want to have an harmonized look within the experimentitemcollection independent of the card content (at least for now)

Reply:
That is however not a property of the card then but of the specific extension of the card?